src/Api/AdminConsole/Controllers/PoliciesController.cs

```diff
@@ -81,7 +112,30 @@ public async Task<ListResponseModel<PolicyResponseModel>> Get(string orgId)
         }
 
         var policies = await _policyRepository.GetManyByOrganizationIdAsync(orgIdGuid);
-        var responses = policies.Select(p => new PolicyResponseModel(p));
```

Comment:
Correct me if I'm overlooking something, but I can't see that we need this property on this endpoint. The policy add-edit dialog only uses the single GET endpoint. Each policy validator can make arbitrary database calls, so I don't want to call them in a loop here unnecessarily.

I suggest splitting the response models so that the single GET endpoint is returning an extended response model containing the new property, and this endpoint is unchanged.